src/unstract/sdk/index.py

```diff
@@ -246,9 +250,29 @@ def index(
                     x2text = X2Text(
                         tool=self.tool, adapter_instance_id=x2text_instance_id
                     )
-                    extracted_text = x2text.process(
-                        input_file_path=file_path, output_file_path=output_file_path
-                    )
+                    if enable_highlight and isinstance(
+                        x2text._x2text_instance, LLMWhisperer
+                    ):
+                        process_response: TextExtractionResult = x2text.process(
+                            input_file_path=file_path,
+                            output_file_path=output_file_path,
+                            enable_highlight=enable_highlight,
+                        )
+                        whisper_hash_value = (
+                            process_response.extraction_metadata.whisper_hash
```

Comment:
For Unstructured Community/Enterprise,  extraction_metadata will be None and accessing whisper_hash from None will raise an exception @johnyrahul